3.11.4 - Cannot find module mediasoup/node/lib/index.js

Hi,

We're using Mediasoup's NodeJS client, and there seems to be a regression in tag 3.11.4. Starting our node application fails with: 

```
Error: Cannot find module '/service/node_modules/mediasoup/node/lib/index.js'. Please verify that the package.json has a valid "main" entry
    at tryPackage (internal/modules/cjs/loader.js:321:19)
    at Function.Module._findPath (internal/modules/cjs/loader.js:534:18)
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:888:27)
    at Function.Module._load (internal/modules/cjs/loader.js:746:27)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:93:18)
    at Object.<anonymous> (/service/server.js:17:19)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32) {
  code: 'MODULE_NOT_FOUND',
  path: '/service/node_modules/mediasoup/package.json',
  requestPath: 'mediasoup'
```

With 3.11.3 this doesn't happen.

Comparing the 2 tags, `node/lib/index.js` indeed seems to have been removed but it's still referenced by `package.json` https://github.com/versatica/mediasoup/compare/3.11.3..3.11.4

Is this indeed a regression, or rather something we should fix in our NodeJS app?

Kind regards,
Mark

I did one final test: installing with npm 6 and npm 8 at the same time, in separate folders.

* NPM 6 mentions it will execute the prepare script, then runs postinstall which ends up running `make` - not sure if this is normal. This takes a long time and finally fails on missing x86-64 symbols (probably because I'm on arm64).
* NPM 8 oddly does not mention the prepare script, but does mention the postinstall, and finishes successfully (with the `node/lib/` directory present) - in about half the time that NPM 6 takes. I'm not sure if NPM 8 also ran `make`.

Partial output below. Probably not useful for this old node/npm version, but yeah :) 

Node 14 / npm 6
```
Ξ js/mediasoup-node14 → npm install versatica/mediasoup#3.11.4 --verbose
npm info it worked if it ends with ok
npm verb cli [
npm verb cli   '/Users/mark/.nvm/versions/node/v14.19.0/bin/node',
npm verb cli   '/Users/mark/.nvm/versions/node/v14.19.0/bin/npm',
npm verb cli   'install',
npm verb cli   'versatica/mediasoup#3.11.4',
npm verb cli   '--verbose'
npm verb cli ]
npm info using npm@6.14.16
npm info using node@v14.19.0
npm verb npm-session afe9a3dd8e02ffbf
npm verb prepareGitDep github:versatica/mediasoup#3.11.4: installing devDeps and running prepare script.
npm ERR! prepareGitDep 1>
npm ERR! prepareGitDep > mediasoup@3.11.4 postinstall /Users/mark/.npm/_cacache/tmp/git-clone-13b07498
npm ERR! prepareGitDep > node npm-scripts.js postinstall
npm ERR! prepareGitDep
npm ERR! prepareGitDep npm-scripts.js [INFO] running task "postinstall"
npm ERR! prepareGitDep npm-scripts.js [INFO] buildWorker()
npm ERR! prepareGitDep npm-scripts.js [INFO] executeCmd(): make -C worker
...
npm ERR! prepareGitDep       RTC::DtlsTransport::GenerateCertificateAndPrivateKey() in src_RTC_DtlsTransport.cpp.o
npm ERR! prepareGitDep ld: symbol(s) not found for architecture x86_64
npm ERR! prepareGitDep clang: error: linker command failed with exit code 1 (use -v to see invocation)
npm ERR! prepareGitDep ninja: build stopped: subcommand failed.
...
```

Node 16 / npm 8
```
Ξ js/mediasoup-node16 → npm install versatica/mediasoup#3.11.4 --verbose
npm verb cli [
npm verb cli   '/Users/mark/.nvm/versions/node/v16.14.0/bin/node',
npm verb cli   '/Users/mark/.nvm/versions/node/v16.14.0/bin/npm',
npm verb cli   'install',
npm verb cli   'versatica/mediasoup#3.11.4',
npm verb cli   '--verbose'
npm verb cli ]
npm info using npm@8.3.1
npm info using node@v16.14.0
npm timing npm:load:whichnode Completed in 0ms
...
npm timing build:link Completed in 317ms
npm info run mediasoup@3.11.4 postinstall node_modules/mediasoup node npm-scripts.js postinstall
npm info run mediasoup@3.11.4 postinstall { code: 0, signal: null }
npm timing build:run:postinstall:node_modules/mediasoup Completed in 71505ms
npm timing build:run:postinstall Completed in 71506ms
...
```

--- a/.github/workflows/mediasoup-node.yaml
+++ b/.github/workflows/mediasoup-node.yaml
@@ -9,8 +9,6 @@ jobs:
         # Different Node versions on Ubuntu, the latest Node on other platforms
         ci:
           - os: ubuntu-22.04
-            node: 14
-          - os: ubuntu-22.04
             node: 16
           - os: ubuntu-22.04
             node: 18

--- a/package.json
+++ b/package.json
@@ -45,7 +45,7 @@
     "nodejs"
   ],
   "engines": {
-    "node": ">=12"
+    "node": ">=16"
   },
   "scripts": {
     "prepare": "node npm-scripts.js prepare",

